sequence_test.go

```diff
@@ -674,15 +674,17 @@ func TestRange(t *testing.T) {
 //===========================================================================
 
 func BenchmarkSequence(b *testing.B) {
+	var s uint64
+	f := func(u uint64) {}
 
+	seq, err := New()
+	if err != nil {
+		b.Error(err.Error())
+	}
+	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		seq, err := New()
```

Comment:
The intent here wasn't to benchmark a single call to `Next()` - though I see how that can be valuable; in my use case, we needed to scan quickly through a large range several times, so it was to benchmark the performance of unbounded monotonically increasing indexes.  

This update is fine, what I'll do in the near future is to have a `BenchmarkSequence` and `BenchmarkSequenceNext` to ensure both cases are covered.

Reply:
Oh, I see! I just thought the test was slow and did not understand the use case - that was the reason that I changed it. But yes, in your use case it makes a complete sense and it will be more clear when you have two different test cases.

Reply:
Cool - #2  will resolve this! Thanks again!